frontend/src/lib/utils/constants.ts

```diff
@@ -1,4 +1,6 @@
 export const constants: Record<string, string> = {
   BACKEND_URL: import.meta.env.VITE_BACKEND_URL || process.env.VITE_BACKEND_URL || '',
+  PUBLIC_BACKEND_URL:
+    import.meta.env.VITE_PUBLIC_BACKEND_URL || process.env.VITE_PUBLIC_BACKEND_URL || '',
```

Comment:
TODO: Before merging this, get changes from PR #267 and implement them here properly as well to get the image urls working.